apps/portal/app/components/list/claims.tsx

```diff
@@ -165,7 +166,9 @@ export function ClaimsListNew({
                   id: triple.subject?.id,
                   description: '',
                   ipfsLink: '',
-                  link: '',
+                  // TODO: ENG-4782 Replace any with correct type
+                  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+                  link: getAtomLinkGQL(triple.subject as any, readOnly),
```

Comment:
I think until we have our types locked in, we can cast as `triple.subject as Atom` with Atom being the type we pull from GraphQL. Pretty sure we've done this elsewhere -- I remember casting the triple.subject as Atom before.

Which reminds me, we should also create a ticket to move all of the new type exports into a single place since we've been doing them in each file where it's used.